package-lock.json

```diff
@@ -9,6 +9,7 @@
       "version": "0.1.0",
       "license": "MIT",
       "dependencies": {
+        "@datastructures-js/priority-queue": "^6.3.0",
```

Comment:
Nice find! Whenever I need a pq in js, I always end up copying from some medium post lol.